crates/bevy-inspector-egui/src/inspector_options/default_options.rs

```diff
@@ -114,17 +193,20 @@ pub fn register_default_options(type_registry: &mut TypeRegistry) {
                 ("radius", &NumberOptions::<f32>::positive()),
             ],
         );
+
+        #[rustfmt::skip]
         insert_options_struct::<bevy_pbr::DirectionalLight>(
             type_registry,
-            &[("illuminance", &NumberOptions::<f32>::positive())],
+            &[
+                ("illuminance", &NumberOptions::<f32>::positive())
+            ],
```

Comment:
If you're gonna do `#[rustfmt::skip]`, then put this in a single line.

Reply:
The skips were meant to make the code more uniform, making it easier to copy paste and do multiple cursors. I think I'll just remove them instead.

Reply:
Or actually, the formatted ones are so much worse xD I'll keep the skips and make them as uniform as possible